lms/djangoapps/courseware/tests/test_views.py

```diff
@@ -307,6 +307,64 @@ def test_index_no_visible_section_in_chapter(self):
         self.assertNotIn('Problem 1', response.content)
         self.assertNotIn('Problem 2', response.content)
 
+    def _create_url_for_enroll_staff(self):
```

Comment:
This method is doing two things:
1) creating a staff user and a course they're not enrolled in
2) creating a courseware and enroll_staff url

These should be separated out into two functions for readability and clearer separations of concern